src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketTeamMetadataAction.java

```diff
@@ -88,6 +88,7 @@ public AvatarImage fetch() {
 
         @Override
         public String hashKey() {
+            StandardCredentials credentials = bbCredentials.credentials();
```

Comment:
https://github.com/jenkinsci/bitbucket-branch-source-plugin/issues/331#issuecomment-745179777
It looks like theres are problem on the next line with an NPE.